tests/test_views.py

```diff
@@ -1,15 +1,47 @@
 import pytest
+from rest_framework import status
 
 
 def test_api_parse_succeeds(client):
     # TODO: Finish this test. Send a request to the API and confirm that the
     # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+    response = client.get(
+        "/api/parse/",
+        {"address": address_string},
+    )
+    assert response.status_code == 200
+
+    expected_dict = dict({
+        "input_string": address_string,
+        "address_components": {
+            "AddressNumber": "123",
+            "StreetName": "main",
+            "StreetNamePostType": "st",
+            "PlaceName": "chicago",
+            "StateName": "il"
+        },
+        "address_type": "Street Address"
+    })
+
+    if response.json()['data'] is None:
+        pytest.fail()
```

Comment:
Why check like this, rather than using an assertion as in your other checks?

Reply:
Very good point. I'm not sure why I didn't just use `assert` but the change has been made in both tests.